src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -5592,7 +5598,90 @@ GUI_TEST_CLASS_DEFINITION(test_6754) {
+{
+    GTLogTracer l;
+
+    //    1. Open sequence
+    //    2. Open annotation file
+    //    3. Add annotation file to sequence
+    //    4. Find splitter
+    //    5. Find scroll in splitter
+    //    6. Scroll to end, then to begin
+
+    GTFileDialog::openFileList(os,
+                               testDir + "_common_data/regression/6759/",
+                               QStringList() << "annotations.gb"
+                                             << "sequence.gb");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+    CHECK_SET_ERR(GTUtilsProjectTreeView::checkItem(os, "Unknown features"),
+                  "No 'Unknown features' object!");
+
+    //    Use context menu on annotation in tree view
+    GTUtilsDialog::waitForDialog(os,
+                                 new ProjectTreeItemSelectorDialogFiller(os,
+                                                                         "annotations.gb",
+                                                                         "Unknown features"));
+    GTUtilsDialog::waitForDialog(os, new CreateObjectRelationDialogFiller(os));
+    GTUtilsDialog::waitForDialog(os,
+                                 new PopupChooserByText(os,
+                                                        QStringList()
+                                                            << "Add"
+                                                            << "Objects with annotations..."));
+    //    On question "Found annotations that are out of sequence range, continue?" answer "Yes"
+    GTUtilsDialog::waitForDialog(os, new MessageBoxDialogFiller(os, QMessageBox::Yes));
+
+    GTUtilsSequenceView::openPopupMenuOnSequenceViewArea(os);
+    //    Check {add-> Objects with annotations} action
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    int seqNum = GTUtilsSequenceView::getSeqWidgetsNumber(os);
+    CHECK_SET_ERR(seqNum == 1, QString("Too many seqWidgets count").arg(seqNum));
+
+    ADVSingleSequenceWidget *seqWgt = GTUtilsSequenceView::getSeqWidgetByNumber(os, 0);
+    CHECK_SET_ERR(GTUtilsCv::isCvPresent(os, seqWgt), QString("No CV for single sequence view."));
+    ADVSequenceObjectContext *ctx = seqWgt->getActiveSequenceContext();
+    CHECK_SET_ERR(ctx != nullptr, QString("No context for single sequence view."));
+    AnnotatedDNAView *aview = ctx->getAnnotatedDNAView();
+    CHECK_SET_ERR(aview != nullptr, QString("No annoview for single sequence view. First check"));
+    QList<ADVSplitWidget *> splitters = aview->getSplitWidgets();
+    CHECK_SET_ERR(splitters.size() == 1,
+                  QString("No splitters for single sequence view. First check"));
+    CircularViewSplitter *splitter = static_cast<CircularViewSplitter *>(splitters[0]);
+    CHECK_SET_ERR(splitter != nullptr, QString("CircularViewSplitter is null"));
+
+    QList<QScrollBar *> scrolls = splitter->findChildren<QScrollBar *>("CircularViewSplitter_horScroll");
```

Comment:
Use GTWidget::find.. method which will check type and assert that the result exists.

Reply:
done